python/ray/serve/_private/http_proxy.py

```diff
@@ -322,6 +337,14 @@ async def __call__(self, scope, receive, send):
                 scope, receive, send
             )
 
+        # The current autoscale logic can downscale nodes with ongoing requests if the
+        # node doesn't have replicas and has no object references. This counter and
+        # the dummy object reference will have to keep the node alive while draining
+        # requests, so they are not dropped unintentionally.
+        self._ongoing_requests += 1
```

Comment:
Can we add a brief time delay here? If a user has a low QPS, their proxies will constantly be putting and deleting stuff from the object store.

Reply:
I'm not sure if it's a good idea to add a time delay here. It will also make each query longer. Also not sure if it's an issue to keep putting and dropping things in the data store?

Reply:
I don't think this is much of a concern because the overhead of putting a tiny object in the object store should be small and by definition it will only happen when load is low

Reply:
Sounds good